fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/jbatch/bulkdata/load/ChunkWriter.java

```diff
@@ -139,8 +153,18 @@ public void writeItems(List<java.lang.Object> arg0) throws Exception {
         FHIRPersistenceHelper fhirPersistenceHelper = new FHIRPersistenceHelper();
         FHIRPersistence fhirPersistence = fhirPersistenceHelper.getFHIRPersistenceImplementation();
         FHIRPersistenceContext persistenceContext = FHIRPersistenceContextFactory.createPersistenceContext(null);
+
         FHIRTransactionHelper txn = new FHIRTransactionHelper(fhirPersistence.getTransaction());
 
+        // Create a new FHIRRequestContext and set it on the current thread.
+        FHIRRequestContext context = new FHIRRequestContext(fhirTenant, fhirDatastoreId);
+        // Don't try using FHIRConfigHelper before setting the context!
+        FHIRRequestContext.set(context);
+        context.setOriginalRequestUri(incomingUrl);
+        context.setBulk(true);
+
+        logger.fine("The incomingUrl is '" + incomingUrl + "'");
```

Comment:
isLoggable

Reply:
I'll remove it... since I know it exists now...